lib/PPI/Token/Data.pm

```diff
@@ -60,7 +67,14 @@ Returns an L<IO::String> object.
 
 sub handle {
 	my $self = shift;
-	IO::String->new( \$self->{content} );
+	# perl 5.6 compatibility
+	if (USE_IO_STRING) {
+		return IO::String->new( \$self->{content} );
+	}
+	else {
+		open my $fh, '+<', \$self->{content};
+		return $fh;
+	}
```

Comment:
Not critical, but wondering. Why not:
```
	# perl 5.6 compatibility
	return IO::String->new( \$self->{content} )
	  if USE_IO_STRING;

	open my $fh, '+<', \$self->{content};
	return $fh;
```

Reply:
I prefer to think of this as two alternative paths that just happen to include returns, which I think stylistically matches better with if + else, rather than a early escape for one condition, which is implied by the statement modifier.